sites/partners/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "dev": "NODE_OPTIONS='--inspect=9230' next -p ${NEXTJS_PORT:-3001}",
```

Comment:
Does it make sense to flip flop this if 9230 is already defined upstream for partners?

Reply:
I'm going to contribute this upstream as well. I wanted to align the debug ports with the http ports.